docs/modules/ROOT/pages/release-notes/release-notes-28-0.adoc

```diff
@@ -0,0 +1,27 @@
+= Payara Enterprise 5.28.0 Release Notes
+
+== Supported APIs and Applications
+
+* Java EE 8 Applications
+* Jakarta EE 8
+* MicroProfile 3.3
+
```

Comment:
Please add

== Notes

=== Metro (JAX-WS implementation) remote code vulnerability

We have fixed a remote code vulnerability in the Metro framework. If you have an application deployed on the Payara Server that makes use of the JAX-WS features, please update your environment.